ca/src/test/java/com/cletus/app/QueryProcessorTest.java

```diff
@@ -0,0 +1,38 @@
+package com.cletus.app;
+
+import org.junit.jupiter.api.Test;
```

Comment:
I'm getting some errors still with this import.  What is the recommended way to install it?

Thanks!

Reply:
This has been updated, new code does not use any dependencies anymore